doc/sphinx/changes.rst

```diff
@@ -8,6 +8,519 @@ Recent changes
+  :n:`Module F (E:T) := E`
+  (`#15412 <https://github.com/coq/coq/pull/15412>`_,
+  fixes `#15403 <https://github.com/coq/coq/issues/15403>`_,
+  by Hugo Herbelin).
+- **Fixed:**
+  We introduce a new irrelevant term in the reduction machine.
+  It is used to shortcut computation of terms living in a strict
+  proposition, and behaves as an exception. This restores subject
+  reduction, and also makes conversion of large terms in SProp
+  cheap
+  (`#15575 <https://github.com/coq/coq/pull/15575>`_,
+  fixes `#14015 <https://github.com/coq/coq/issues/14015>`_,
+  by Pierre-Marie Pédrot).
+- **Fixed:**
+  performance blowups while inferring variance information for :ref:`cumulative` inductive types
+  (`#15662 <https://github.com/coq/coq/pull/15662>`_,
+  fixes `#11741 <https://github.com/coq/coq/issues/11741>`_,
+  by Gaëtan Gilbert).
+
+Specification language, type inference
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Added:**
+  New clause :n:`as @ident` to the :cmd:`Record` command to specify
+  the name of the main argument to use by default in the type of
+  projections
+  (`#14563 <https://github.com/coq/coq/pull/14563>`_,
+  by Hugo Herbelin).
+- **Added:**
+  :term:`Reversible coercions <reversible coercion>` are coercions which cannot be
+  represented by a regular coercion (a Gallina function)
+  but rather a meta procedure, such as type class inference
+  or canonical structure resolution
+  (`#15693 <https://github.com/coq/coq/pull/15693>`_,
+  by Cyril Cohen, Pierre Roux, Enrico Tassi,
+  reviewed by Ali Caglayan, Jim Fehrle and Gaëtan Gilbert).
+- **Added:**
+  support for coercions not fulfilling
+  the uniform inheritance condition,
+  allowing more freedom for the parameters that are now inferred
+  using unification, canonical structures or typeclasses
+  (`#15789 <https://github.com/coq/coq/pull/15789>`_,
+  fixes `#2828 <https://github.com/coq/coq/issues/2828>`_,
+  `#4593 <https://github.com/coq/coq/issues/4593>`_,
+  `#3115 <https://github.com/coq/coq/issues/3115>`_,
+  `#5222 <https://github.com/coq/coq/issues/5222>`_,
+  `#9696 <https://github.com/coq/coq/issues/9696>`_
+  and `#8540 <https://github.com/coq/coq/issues/8540>`_,
+  by Pierre Roux, reviewed by Ali Caglayan, Enrico Tassi, Kazuhiko Sakaguchi and Jim Fehrle).
+- **Fixed:**
+  interpretation of `{struct}` fixpoint annotations when the principal argument comes from an implicit generalization
+  (`#15581 <https://github.com/coq/coq/pull/15581>`_,
+  fixes `#13157 <https://github.com/coq/coq/issues/13157>`_,
+  by Gaëtan Gilbert).
+
+Notations
+^^^^^^^^^
+
+- **Removed:**
+  ``_`` in ``ident`` entries in notations, which was deprecated
+  in favor of ``name`` in 8.13. When you see messages like
+  ``Error: Notation "[ rel _ _ : _ | _ ]" is already defined at level 0
+  with arguments name, name, constr, constr while it is now required to be
+  at level 0 with arguments ident, ident, constr, constr.``,
```

Comment:
This long error message is shown entirely on a single line: https://coq.gitlab.io/-/coq/-/jobs/2501178726/artifacts/_install_ci/share/doc/coq/sphinx/html/changes.html#id28

We should find a way of making it break (or manually break it) on several lines.

Reply:
No idea about how to do that.

Reply:
I pushed a commit to try that. Let's see.

Reply:
Seems to have worked: https://coq.gitlab.io/-/coq/-/jobs/2522451683/artifacts/_install_ci/share/doc/coq/sphinx/html/changes.html#id30